ArmRobot/Assets/Scripts/Camera/RandomRotationCamera.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class RandomRotationCamera : MonoBehaviour
```

Comment:
It is good that you are being so modular, but this could probably be combined with `RandomPositionCamera` to make a single `CameraPoseRandomizer` or something like that.

Reply:
Yes you are right, I will gather everything in a same script I will call RandomizerPositionCamera